src/Cryptol/REPL/Command.hs

```diff
@@ -1166,15 +1177,34 @@ typeOfCmd str pos fnm = do
 
   -- XXX need more warnings from the module system
   whenDebug (rPutStrLn (dump def))
+
```

Comment:
I think there might be a more direct way to do this.   I am thinking like this: `getFocusedEnv` is a `ModContext`, which has a `NamingEnv` inside it.  You can use `toNameDisp` to convert this to a `NameDisp`, and put this inside the `PPCfg`.  Then, I think just doing `pp` on the `Schema` should print it correctly.